### PR TITLE
Complete continuous integration

### LINKED ISSUE #48: Create tests for data/simpy.gram
We should add automated tests (and run them on Travis-CI, see `.travis.yml`) that verify that `data/simpy.gram` can parse all of the CPython standard library code.

If there is a CPython checkout at `$CPYTHON`, then this test would be equivalent to running
```
make simpy TESTDIR=$CPYTHON/Lib
```

If getting a CPython checkout in Travis is too slow, we could create a gzipped tarball of just `$CPYTHON/Lib` and check that into the pegen repo, and as part of the test untar it and then run something like the above `make` command.

### PR DESCRIPTION
**Pull request by apalala**
closes #48 

* verify Python package requirements
* clean up `.travis-ci.yml`
* include test for parse coverage of `cpython`
* possibly move testing to `tox` so it can be checked on the command-line (integrate with `tox-travis`)

### CODE REVIEW

File: requirements-dev.pip [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-r requirements.pip
+coveralls==1.8.2
+ipython==7.8.0
+mypy==0.730
+pytest-cov==2.8.1
+pytest==5.2.1
+typing-extensions==3.7.4
```
[the remark this answers is not in the document]
Comment (apalala) [reply requirements-dev.pip]: Fixed!

**Comment by apalala** [comment]
This seems ready to merge. Now each commit will check for parse coverage.

Next steps:
1.  Implement actions to generate AST
2. Make `test_parse_directory.py` compare AST to those of `ast`
3. Add third-party libraries to the coverage tests

**Comment by apalala** [comment]
`ipython` is not needed for testing. Perhaps there should be a separate `requirements-test.pip`, or a `.gitignore` for `requirements-local.pip`?

**Comment by apalala** [comment]
And what's the protocol for merging pull requests?

* Who does the merge?
* How many approvals are required?

**Comment by gvanrossum** [comment]
I will do all merges, until we have some more experience collaborating. Only one approval is needed. The GitHub repo should be configured to squash commits always.

**Pull request merged**